src/c++/library/http_client.cc

```diff
@@ -980,6 +981,25 @@ InferResultHttp::IsNullResponse(bool* is_null_response) const
   return Error::Success;
 }
 
+Error
+InferResultHttp::Output(std::string& output) const
+{
+  // Only supports LLM outputs with name 'text_output' currently
+  if (output_name_to_buffer_map_.find("text_output") ==
```

Comment:
Double checking: this == is intentional?
Because this http client does not support LLMs?

Reply:
This is just exiting early if the model has no output named "text_output", leaving the `output` arg unmodified - which seems expected to me for now for Triton-based clients of Triton LLM backends.